Makefile.am

```diff
@@ -190,6 +190,7 @@ STATIC_TESTS = \
 	tests/test_jid \
 	tests/test_ctx \
 	tests/test_send_queue \
+	tests/test_serialize_sm \
```

Comment:
Did you by an hazard already create tests for this or was this only a stub?

Reply:
good catch I forgot to add the file. added now